harvest/broker/dummy.py

```diff
@@ -59,9 +62,43 @@ def setup(self, watch: List[str], interval, trader=None, trader_main=None):
         super().setup(watch, interval, interval, trader, trader_main)
 
     @base.BaseBroker.main_wrap
-    def main(self) -> pd.DataFrame:
-        results = None
-        return results
+    def main(self):
+        df_dict = {}
+        df_dict.update(self.fetch_latest_stock_price())
+        df_dict.update(self.fetch_latest_crypto_price())
+      
+        return df_dict
+
+        # if self.mode in ('both', 'streamer'):
```

Comment:
This is what I hope the main will look like in the future. The streamer part will work just fine, but the broker one will not since it wants to send all the orders on interval.